studies/armCartesianSpaceSquare/src/main/java/frc/robot/subSystems/FRCNEO.java

```diff
@@ -1,4 +1,4 @@
-package frc.robot;
+package frc.robot.subSystems;
```

Comment:
package names should be all lower case.  also if you wanted to follow the lib example this would be in package named something like org.team100.cartesiansquare.motion.arm